File: frontend/react-app/src/api/internal/dto/PresenceDto.js
export default class PresenceDto {
    constructor(individualId, isOnline, lastOnlineDate) {
        this.individualId = individualId;
        this.isOnline = isOnline;
        this.lastOnlineDate = lastOnlineDate;
    }

    static fromJSON(data) {
        return new PresenceDto(
            data.individualId,
            data.isOnline,
            new Date(data.lastOnlineDate)
        )
    }
}

File: backend/Backend/src/main/java/ru/nikidzawa/backend/store/client/dto/PresenceDto.java
package ru.nikidzawa.backend.store.client.dto;

import lombok.*;
import lombok.experimental.FieldDefaults;

import java.time.LocalDateTime;

/**
 * @author Nikidzawa
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class PresenceDto {
    Long individualId;
    Boolean isOnline;
    LocalDateTime lastOnlineDate;
}
